### ISSUE TITLE
Graphics window not saving

### ISSUE THREAD

**Issue by jmjech** [spyder-ide/spyder#19071]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I just updated anaconda and spyder to 5.2.2 as per the spyder update instructions and now the graphics window will not save the image. I have the graphics backend set to Automatic so that plots are displayed in separate windows. When I click on the save icon in the graphics window, the pickfile dialog box flashes on the screen and closes, and the graphics window essentially freezes and will not allow me to close it using the "x" in the upper right corner. Before updating, I didn't have this problem.

### What steps reproduce the problem?

1. graphics backend set to Automatic
2. any matplotlib plot command
3. 

### What is the expected output? What do you see instead?
I expect to be able to select a file to save to. Now the pickfile dialog box flashes on the screen, not allowing me to select a file.

### Paste Traceback/Error Below (if applicable)

```python-traceback

no traceback is given

```

## Versions

* Spyder version: 5.2.2
* Python version: 3.8
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Linux 5.15.0-46-generic, Ubuntu 20.04.4 LTS

Python 3.8.12 64-bit | Qt 5.9.7 | PyQt5 5.9.2 | Linux 5.15.0-46-generic 

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.4.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  61.2.0 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  0.29.30 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.4.3 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @jmjech, thanks for reporting. I think this is fixed in Matplotlib 3.5.2.

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

Please try that and let us know if it works for you.

**Comment by jmjech** [comment]
Hi @ccordoba12 ,
That did not fix this issue. When I used your commands, matplotlib v3.5.1 was installed and the figure-saving issue was not fixed. I then installed matplotlib v3.5.3, and am having the same issue. In addition, I needed to install other packages, and most were successful, but scipy did not install correctly and I need to figure out why that is.

It may be that I go back to the version that anaconda has and use that until anaconda upgrades. It seems anaconda is not on the bleeding edge, but they do seem to get all the dependencies in order.

Thanks,
@jmjech
